example-schema/types/MainType.yaml

```diff
@@ -30,6 +35,11 @@ properties:
     type: AnEnum
     description: Enum description.
     label: Enum label
+  someMultipleChoice:
+    type: AnEnum
+    hasMany: true
+    description: Multiple choice description
+    label: Multiple choice label
```

Comment:
Enum is suggestive `allowed fixed number` to me but the tests are actually using a string like 'First', 'Second'. Are these OK for us?  I can't find the actual value choice lists which this schema accepts on other place.

So I assume `AnEnum` field should have a field that declares user choosable values like `choise: ['First', 'Second']`

Reply:
The enums defined in the graphql api map directly to string representations of them stored in the database. I don't love how enums work in graphQL to be honest, but this is how it is, and it's consistent with other uses of enum. From wikipedia:

- a data type consisting of a set of named values called elements, members, enumeral, or enumerators of the type. The enumerator names are usually identifiers that behave as constants in the language